x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/overview/alerts.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiFlexGroup, EuiFlexItem, type EuiAccordionProps } from '@elastic/eui';
+import type { TimeRange } from '@kbn/es-query';
+import type { InventoryItemType } from '@kbn/metrics-data-access-plugin/common';
+import { findInventoryFields } from '@kbn/metrics-data-access-plugin/common';
+import { usePluginConfig } from '../../../../containers/plugin_config_context';
+import { AlertFlyout } from '../../../../alerting/inventory/components/alert_flyout';
+import { useBoolean } from '../../../../hooks/use_boolean';
+import { AlertsSectionTitle } from '../../components/section_titles';
+import { useAssetDetailsRenderPropsContext } from '../../hooks/use_asset_details_render_props';
+import { CollapsibleSection } from './section/collapsible_section';
+import { AlertsClosedContent } from './alerts_closed_content';
+import { type AlertsCount } from '../../../../hooks/use_alerts_count';
+import { AlertsOverview } from '../../../shared/alerts/alerts_overview';
+import { LinkToAlertsRule } from '../../../shared/alerts/links/link_to_alerts';
+import { LinkToAlertsHomePage } from '../../../shared/alerts/links/link_to_alerts_page';
+
+export const AlertsSummaryContent = ({
+  assetName,
+  assetType,
+  dateRange,
+}: {
+  assetName: string;
+  assetType: InventoryItemType;
+  dateRange: TimeRange;
+}) => {
+  const { featureFlags } = usePluginConfig();
+  const [isAlertFlyoutVisible, { toggle: toggleAlertFlyout }] = useBoolean(false);
+  const { overrides } = useAssetDetailsRenderPropsContext();
+  const [collapsibleStatus, setCollapsibleStatus] =
+    useState<EuiAccordionProps['forceState']>('open');
+  const [activeAlertsCount, setActiveAlertsCount] = useState<number | undefined>(undefined);
+
+  const onLoaded = (alertsCount?: AlertsCount) => {
+    const { activeAlertCount = 0 } = alertsCount ?? {};
+    const hasActiveAlerts = activeAlertCount > 0;
+
+    setCollapsibleStatus(hasActiveAlerts ? 'open' : 'closed');
+    setActiveAlertsCount(alertsCount?.activeAlertCount);
```

Comment:
You could remove the `collapsibleStatus` state and use the `activeAlertsCount` to define the accordion status.